source/store.go

```diff
@@ -277,7 +277,7 @@ func BuildWithConfig(source string, p ClientGenerator, cfg *Config) (Source, err
 		if err != nil {
 			return nil, err
 		}
-		return NewCRDSource(crdClient, cfg.Namespace, cfg.CRDSourceKind, cfg.AnnotationFilter, cfg.LabelFilter, scheme)
+		return NewCRDSource(crdClient, cfg.Namespace, cfg.CRDSourceKind, cfg.AnnotationFilter, cfg.LabelFilter, scheme, true)
```

Comment:
I would appreciate if we could change true to a meaningful var otherwise no one will remember what it is and we need to look up

Reply:
Good idea; just pushed a commit with that change.  Thanks!